src/components/ActivityTitle/styles.js

```diff
@@ -29,29 +20,60 @@ export const ActivityTitleText = styled.h1`
 
 export const RankingWrap = styled.div`
   display: flex;
+  width: 100%;
   align-items: center;
+`;
 
-  @media ${device.deviceL} {
-    /* width: 100vw;
-    height: 100px; */
-    transition: all 0.2s ease-in-out;
+export const CountryTitle = styled.a`
+  text-decoration: none;
+  text-decoration: underline;
+  font-weight: bold;
+  padding-top: 3px;
+
+  right: 70px;
+
+  color: ${colors.darkText};
+  :hover {
+    color: ${colors.darkText};
+    text-decoration: underline;
+  }
+  :focus {
+    color: ${colors.darkText};
+    text-decoration: underline;
+  }
+  :active {
+    color: ${colors.darkText};
+    text-decoration: underline;
+  }
+  :visited {
+    color: ${colors.darkText};
+    text-decoration: underline;
   }
 `;
 
 export const ShareButton = styled.button`
   display: none;
+
   @media ${device.deviceL} {
+    position: absolute;
```

Comment:
non usare questo approccio, dovresti usare flex in modo che sia dinamico. Dovresti racchiudere rank e il titolo in un div 

es.
`div  flex  + justify: space-between
     div
         rank 
         country
     /div
     
    condividi
 /div
`